pkg/server/grpc.go

```diff
@@ -24,8 +25,10 @@ import (
 )
 
 var (
-	MalwareScanDir = "/"
-	HostMountDir   = "/fenced/mnt/host"
+	MalwareScanDir          = "/"
+	HostMountDir            = "/fenced/mnt/host"
+	cntnrPathPrefixRegex    = `.+?\/overlay2\/[A-z0-9a-z]+\/merged`
```

Comment:
IIRC `merged` is not guaranteed, it might be `diff`, `lower` or `work` as per https://pkg.go.dev/github.com/barracks510/go-overlay2